### PR TITLE
Tailwind prettier plugin

### LINKED ISSUE #28: Tailwind prettier preset

### PR DESCRIPTION
**Pull request by kalwabed**
Fix #28

### CODE REVIEW

**Comment by kalwabed** [comment]
Waiting for #27 before merging.

**Review by zeetec20 — APPROVED** [review]

**Comment by kalwabed** [comment]
No, it's not ready for review. Not yet. Please be patient @zeetec20.

It's waiting for #27 to be merged first!

**Comment by zeetec20** [comment]
Okay let me fix and merge first

**Review by zeetec20 — APPROVED** [review]

**Pull request merged**
